### ISSUE TITLE
Need Algorithmic Method to Infer Sidewalk Locations

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#1154]
We need an algorithmic method to infer sidewalk locations and, relatedly, to split up street segments into two sidewalk segments (one for each side of street).

We can use the 'no sidewalk' label to infer when there are _not_ sidewalks... can use the inverse, perhaps, as a starting point to infer where there are sidewalks.

**Comment by jonfroehlich** [comment]
@misaugstad and I have discussed this a lot. The larger question isn't just algorithmic methods but also algorithmic + crowdsourcing interfaces. We're not sure that the GSV-based approach (alone) is necessarily the best way to pursue this vs., say, a crowd labeling interface that combines a top-down satellite view (or even a bird's eye view) along with, possibly, the GSV pane for confirmation (and to get around occlusion issues with top-down photographs).

But it feels like a different Project Sidewalk project: something with it's own interfaces/workflows just for labeling the absence/presence of sidewalks. Also relates to OpenSidewalks.com and other initiatives out there (indeed, Kotaro and I have been working on a paper for a while now on this too).

**Comment by misaugstad** [comment]
Folding into #1129, which is the earlier and broader thread on the same problem — where sidewalks are and how we figure that out — and which has kept collecting the relevant material (the Ning et al. aerial+street-view extraction paper, Nearmap, DC's 3" open aerial imagery).

Jon's own comment here points the same way: the open question isn't purely algorithmic, it's what combination of algorithm and crowdsourcing interface to use, which is exactly what #1129 is for. The modern concrete proposals descending from it are #3678 (label assets from satellite imagery) and #3904 (record street-level attributes by clicking the minimap).

Keeping #1155 (attaching labels to sidewalk segments) separate, since that's a different deliverable that depends on having a sidewalk network in the first place.

**Comment by jonfroehlich** [comment]
I'd like to keep this open as just today I was working on this.

**Comment by misaugstad** [comment]
Cool, just try to resolve both from the PR! Just clearing out some tickets where I've seen dupes created. Because we tend to create dupes, then resolve the dupes without getting back to these old ones

**Comment by jonfroehlich** [comment]
Yes. For sure. Love that you’re doing this. Thanks for tidying up!
